index.js

```diff
@@ -41,7 +41,11 @@ function (createConnection) {
 
       function onError (err) {
         con.removeListener('error', onError)
-        emitter.emit('error', err)
+        try
+        {
+          emitter.emit('error', err)
+        }
+        catch(e){}
```

Comment:
i won't merge this, users _need_ to listen to the `"error"` event. if this isn't a real error, i'd rather rename to `"warning"` in stead of bypassing standard node core logic.